pkg/cmd/roachtest/tests/ycsb.go

```diff
@@ -89,15 +87,20 @@ func registerYCSB(r registry.Registry) {
 		m.Go(func(ctx context.Context) error {
 			var args string
 			args += " --ramp=" + ifLocal(c, "0s", "2m")
-			args += " --duration=" + ifLocal(c, "10s", "30m")
 			if opts.readCommitted {
 				args += " --isolation-level=read_committed"
 			}
 			if opts.uniformDistribution {
 				args += " --request-distribution=uniform"
 			}
-			if envFlags := os.Getenv(envYCSBFlags); envFlags != "" {
-				args += " " + envFlags
+
+			envWorkloadDurationFlag := os.Getenv(test.EnvWorkloadDurationFlag)
+			defaultDuration := ifLocal(c, "10s", "30m")
+
+			if envWorkloadDurationFlag != "" && test.ValidateWorkloadDurationFlag(envWorkloadDurationFlag) {
```

Comment:
This block is identical to the one above. Maybe lift this into a helper function (inside `utils.go`)?

Reply:
Done.